helpers/common.go

```diff
@@ -2,19 +2,24 @@ package helpers
 
 import (
 	"fmt"
+	"github.com/companieshouse/chs-delta-api/config"
 	"github.com/companieshouse/chs.go/log"
 	"io/ioutil"
 	"net/http"
+	mrand "math/rand"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#crypto-rand

Reply:
Changed it to use crypto/rand library